app/validators/schema_validator.py

```diff
@@ -41,7 +41,7 @@ def validate(self):
             return {}
         except ValidationError as e:
             match = best_match([e])
-            self.add_error(match.message, verbose=e.message, path=e.path)
+            self.add_error(match.message, verbose=e.message, path=str(e.path))
```

Comment:
I think it's more useful to return a string path like: `sections/2/title"` instead of ` "deque(['sections', 2, 'title'])",`

Reply:
I've made this a pointer, given it's used in translations and easy to covert to.